docs/source/router.rst

```diff
@@ -0,0 +1,97 @@
+Router
+======
+
+Working with Routers
+````````````````````
+
+Real world applications can almost never fit all logic into a single file.
+As Flask supports splitting your API into smaller modules via Blueprints, in FlaskNinja the module is called Router.
+
+When you create a new Router instance, you can register your endpoints there as to the Ninja API instance:
+
+``blogs.py``
+
+.. code-block:: python
+
+    from flask_ninja.router import Router
+
+    api = Router()
+
+    @api.get("/")
+    def get_blogs() -> Response:
+        ...
+
+    @api.post("/new")
+    def create_blog() -> Response:
+        ...
+
+
+``users.py``
+
+.. code-block:: python
+
+    from flask_ninja.router import Router
+
+    api = Router()
+
+    @api.get("/")
+    def get_users() -> Response:
+        ...
+
+    @api.post("/new")
+    def create_user() -> Response:
+        ...
+
+
+Then you combine all endpoints by adding the routers to the API. You can also add a prefix to endpoints of a router,
```

Comment:
I am not sure about naming here, but if one module is called Router (with R), then when referring to the multiple modules you want to keep the capital R to differentiate it from a "router" that we know as the device for wifi functioning :D

Reply:
I'm not aware about confusion with wifi router :) Also other frameworks as django-ninja use it like that